tests/solana.rs

```diff
@@ -1594,17 +1594,19 @@ impl VirtualMachine {
         res
     }
 
-    fn constructor(&mut self, name: &str, args: &[BorshToken]) {
-        self.constructor_expected(0, name, args)
+    /// FIXME: remove name argument
+    fn constructor(&mut self, _name: &str, args: &[BorshToken]) {
```

Comment:
The name argument should be removed but I am worried about 1000s of conflicts with @LucasSte PR https://github.com/hyperledger/solang/pull/1104

We can resolve this later.

Reply:
What happened with the Sean Young that told me he can solve all conflicts in 4 minutes? 🤣🤣🤣

In addition, I have not changed the constructor function.

Reply:
Well, you've changed a lot of lines near the constructor function callsites, so you'll get conflicts.

Reply:
I am happy to fix this now *if* my PR goes in first

Reply:
>  _if_ my PR goes in first

This is an exacting requirement.